### DISCUSSION TITLE
Welcome to RESTPresenter Discussions!

### DISCUSSION THREAD

**Discussion by adam-code-labx** [xtend-packages/rest-presenter#25] [Announcements]
## 👋 Welcome to RESTPresenter Discussions!
  Welcome to our community space! This is the perfect place to:
  * Ask any questions you have about RESTPresenter.
  * Share your ideas for new features or improvements.
  * Engage with other members of the RESTPresenter community.
  * Welcome newcomers and be open-minded to different perspectives. Together, we build a stronger community 💪.

  To get started, introduce yourself below and let us know how you're involved with RESTPresenter!

**Comment by planetarquasar** [comment]
Hello, when trying to see what's going on the  `/rest-presenter` endpoint it throws an error: `Route [login] not defined.`. What am I missing?
